sdk/formrecognizer/ai-form-recognizer/samples/typescript/src/recognizeBusinessCard.ts

```diff
@@ -77,6 +77,10 @@ export async function main() {
  * processing.
  */
 function printSimpleArrayField(businessCard: RecognizedForm, fieldName: string) {
+  if (businessCard.fields[fieldName] == null) {
```

Comment:
Shouldn't be necessary in this TS sample due to the use of nullish coalescing on L80.

Reply:
Removed!